ci/install_deps.sh

```diff
@@ -2,6 +2,7 @@ export RUSTUP_HOME=/usr/local/rustup
 export CARGO_HOME=/usr/local/cargo
 export PATH=/usr/local/cargo/bin:$PATH
 export RUST_VERSION=1.46.0
+export DESIRED_VERSION=3
```

Comment:
I think this needs to be an exact tag, not just the major version.

```suggestion
export DESIRED_VERSION=v3.5.4
```

You can see that https://get.helm.sh/helm-v3.5.4-linux-amd64.tar.gz is a working download URL.